script/strategies/aave-v3/BaseDeployStrategies.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { BaseDeploy } from "../../BaseDeploy.sol";
+import { IEarnVault } from "@balmy/earn-core/interfaces/IEarnVault.sol";
+import { IGlobalEarnRegistry } from "src/interfaces/IGlobalEarnRegistry.sol";
+import {
+  AaveV3StrategyFactory,
+  AaveV3Strategy,
+  IAaveV3Pool,
+  IAaveV3Rewards,
+  IAToken,
+  AaveV3StrategyData
+} from "src/strategies/instances/aave-v3/AaveV3StrategyFactory.sol";
+import { IEarnBalmyStrategy } from "src/interfaces/IEarnBalmyStrategy.sol";
+import { StrategyId } from "@balmy/earn-core/interfaces/IEarnStrategy.sol";
+import { DeployVault } from "../../DeployVault.sol";
+import { console2 } from "forge-std/console2.sol";
+
+contract BaseDeployStrategies is BaseDeploy {
+  function setUp() public override {
+    super.setUp();
+    // Assume vault is already deployed
+    DeployVault deployVault = new DeployVault();
+    deployVault.run();
+  }
+
+  function deployAaveV3Strategy(
+    address aaveV3Pool,
+    address aaveV3Rewards,
+    address aToken,
+    string memory symbol,
```

Comment:
I think we can get the symbol by doing something like `aToken.underlying().symbol()`